docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -362,9 +367,18 @@ Incorrectly formatted override files may be accepted without warnings or errors
            at weblogic.management.configuration.DomainMBeanImpl.setJDBCSystemResources(DomainMBeanImpl.java:11444)
            ...
          ```
-  * Look in your `DOMAIN_HOME/optconfig` directory.
-      * This directory, or a subdirectory within this directory, should contain each of your custom situational configuration files.
-      * If it doesn't, then this likely indicates your domain resource `configOverrides` was not set to match your custom override configuration map name, or that your custom override configuration map does not contain your override files.
+
+* If the Administration Server pod does start but fails to reach ready state or tries to restart:
+  * Check for this message ` WebLogic server failed to start due to missing or invalid situational configuration files` in the Administration Server pod's `kubectl log`
+    * This suggests that the Administration Server failure to start may have been caused by errors found in a configuration override file.
+      * Lines containing the String `situational` may be found in the Administration Server pod log to provide more hints. 
+      * For example:
+        * `<Jun 20, 2019 3:48:45 AM GMT> <Warning> <Management> <BEA-141323> <The situational config file has an invalid format, it is being ignored: XMLSituationalConfigFile[/shared/domains/domain1/optconfig/jdbc/testDS-0527-jdbc-situational-config.xml] because org.xml.sax.SAXParseException; lineNumber: 8; columnNumber: 3; The element type "jdbc:jdbc-driver-params" must be terminated by the matching end-tag "</jdbc:jdbc-driver-params>".`
+      * The warning message suggests a syntax error is found in the provided configuration override file for the testDS JDBC datasource.
+  
+* Look in your `DOMAIN_HOME/optconfig` directory.
+  * This directory, or a subdirectory within this directory, should contain each of your custom situational configuration files.
+  * If it doesn't, then this likely indicates your domain resource `configOverrides` was not set to match your custom override configuration map name, or that your custom override configuration map does not contain your override files.
```

Comment:
I think this bullet still belongs in its original location.

Reply:
E.g. current lines 379-381 are more clear if they stay in their original location...

Reply:
move lines 379-381 back to before line 371?

Reply:
back to being a sub-bullet of the 'If WebLogic pods do start, then:' bullet.

Reply:
moved